_config.yml

```diff
@@ -408,6 +408,10 @@ VersionCodeDesc:
   en: The latest version for this project. For example, 1.0.0.
   fr: La version de ce projet. Par  exemple, 1.0.0.
 
+GitHubAccount:
```

Comment:
Please change the wording to "Information about submitter" since users don't have to have a Github account to submit changes.